style-guide/src/components/layout/layout.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Link } from 'gatsby';
+
+import styles from '../../styles/index.module.scss';
+import logo from '../../assets/logo.svg';
+
+export default ({ children, location, nav = null }) => (
+  <div>
+    <div className={styles.header}>
+      <Link to="/" style={{ marginRight: '60px' }}>
+        <img alt="Opentable logo" src={logo} className={styles['logo']} />
+        <h1>Design Bar</h1>
+      </Link>
+      <ul>
+        <li>
+          <Link to="/" className={styles.headerLink}>
+            Design Tokens
+          </Link>
+        </li>
+        {/* <li>
+          <Link to="/handover-process" className={styles.headerLink}>
+            Design &amp; engineering handover process
+          </Link>
+        </li> */}
```

Comment:
@neha-ot, not sure of the page name, this is a big long but this is how we can add the link